src/functionalTest/java/uk/gov/hmcts/divorce/citizen/CitizenSubmitApplicationTest.java

```diff
@@ -42,4 +43,20 @@ public void shouldPassValidationAndGiveSuccessWhenCaseDataValid() throws IOExcep
             .isEqualTo(json(expectedResponse(RESPONSE)));
     }
 
+    @Test
+    public void shouldPassValidationAndGiveSuccessWhenCaseDataValid2() throws IOException {
+        Map<String, Object> request = caseData(REQUEST);
+        request.put("marriageDate", LocalDate.now().minus(1, YEARS).minus(1, DAYS));
```

Comment:
is there a reason we need to minus 1 year and 1 day for marriage date ?

Reply:
didnt really check the BE but in the FE we have a validation rule where the marriage must be atleast a year to get a divorce.